### PR TITLE
Fixes #201 migrated to Django v1.11

### LINKED ISSUE #201: Migrate to Django version 1.11 and Python 3.6
With latest version of OS, >Ubuntu 16.04, Debian 9 Stretch having Python >= 3.5, Django 1.7.7 is having issues.

```
AttributeError: module 'html.parser' has no attribute 'HTMLParseError'
```
because HTMLParseError is deprecated from Python 3.3 onwards and removed with Python 3.5. Temporary fix of this [not a good option] is to downgrade Python version or a better way is to upgrade Django version to >1.7. Also with Django 1.8 in guardian/admin.py change ```module_name``` to ```model_name``` at all its occurrences.

Migrate portal to newer Django version, 1.11.

**W/ this also move all used packages in Portal to latest and compatible one.** https://github.com/systers/portal/issues/290

### PR DESCRIPTION
**Pull request by bhavya-reddy**
In reference to issue #201 
image

image

### CODE REVIEW

**Comment by bhavya-reddy** [comment]
@MeepyMay @tapasweni-pathak please review

File: requirements/prod.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-django-allauth==0.19.1
-django-braces==1.8.0
-django-cities-light==3.1.2
-django-ckeditor==4.4.8
-django-crispy-forms==1.4.0
-django-guardian==1.2.5
-django-imagekit==3.2.6
-djangocms-admin-style==0.2.5
-psycopg2==2.6
+Django
```
Comment (tapasweni-pathak) [inline requirements/prod.txt]: For everything please specify the version. Without version the latest will be pulled out in Travis and elsewhere which makes things a bit unstable.

File: systers_portal/systers_portal/urls.py [hunk 2]
```diff
@@ -1,21 +1,19 @@
 from django.conf import settings
-from django.conf.urls import patterns, include, url
+from django.conf.urls import *
```
Comment (tapasweni-pathak) [inline systers_portal/systers_portal/urls.py]: Please specify the ones that you are including.

File: systers_portal/systers_portal/settings/base.py [hunk 3]
```diff
@@ -64,26 +64,29 @@
...
+    'BACKEND': 'django.template.backends.django.DjangoTemplates',
+    'DIRS': [os.path.join(BASE_DIR,'templates')],
+    'APP_DIRS': True,
+    'OPTIONS': {
+        'context_processors': [
     'community.context_processors.communities_processor',
-)
+    'django.contrib.auth.context_processors.auth',
+    'django.contrib.messages.context_processors.messages',
+            'django.template.context_processors.debug',
```
Comment (tapasweni-pathak) [inline systers_portal/systers_portal/settings/base.py]: flake8.

File: requirements/prod.txt (hunk 1 above)
Comment (sonakshisaxena1) [reply requirements/prod.txt]: Hey @bhavya-reddy !
Are you still working on it?

**Comment by tapasweni-pathak** [comment]
Please reopen if you plan to work on this.

**Pull request closed without merging**